client/client.go

```diff
@@ -281,21 +280,131 @@ func newHttpClient() *http.Client {
 	return &http.Client{}
 }
 
-func dialOpenSSL(conf Config, network string, addr string) (net.Conn, error) {
-	contextSSL, err := openssl.NewCtx()
-	// probably should consider reusing the context, but then we
-	// have to propagate it with every request.
-	err = contextSSL.LoadVerifyLocations(conf.ServerCert, "")
+func listSystemCertsFound(certDir string) (int, error) {
```

Comment:
@kacf This is supposed to mimic the `x509.SystemCertPool()` functionality in the standard `Go` lib. Except, that it only counts valid certificates in the directory.

Reply:
Suggest renaming to `checkSystemCertsFound` to indicate that it doesn't return or store anything. Otherwise looks good!